templates/base_delete.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  {% block delete-title %}
+    <h1>Delete Entry</h1>
+  {% endblock %}
+
+  {% block text-warning %}
+    <p>Are you sure you want to delete this entry?</p>
+  {% endblock %}
+
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input class="btn btn-outline-danger" type="submit" value="Yes, delete">
```

Comment:
I think that `No` button would be relevant here

Reply:
Thank you for the advice!
I already implemented this in one of the following tasks, so I copied and pasted it here.

Reply:
oh, okay 👍🏻